src/BaselineUtils.cc

```diff
@@ -33,10 +33,100 @@ const int deblend::BaselineUtils<ImagePixelT, MaskPixelT, VariancePixelT>::STRAY
 template <typename ImagePixelT, typename MaskPixelT, typename VariancePixelT>
 const int deblend::BaselineUtils<ImagePixelT, MaskPixelT, VariancePixelT>::STRAYFLUX_TRIM;
 
-static bool span_ptr_compare(PTR(det::Span) sp1, PTR(det::Span) sp2) {
-    return (*sp1 < *sp2);
+static bool span_compare(geom::Span const & sp1,
+                         geom::Span const & sp2) {
+    return (sp1 < sp2);
```

Comment:
There is no copy being done, it is simply being passed by reference now instead of my pointer. Same effect with less *